pyproject.toml

```diff
@@ -72,38 +72,58 @@ build-backend = "poetry.core.masonry.api"
 
 
 [tool.isort]
+py_version = 36
 profile = "black"
 force_single_line = true
-atomic = true
-include_trailing_comma = true
-lines_after_imports = 2
+combine_as_imports = true
 lines_between_types = 1
-use_parentheses = true
+lines_after_imports = 2
 src_paths = ["poetry", "tests"]
-skip_glob = ["*/setup.py"]
-filter_files = true
-known_first_party = "poetry"
+extend_skip = ["setup.py"]
 
 
 [tool.black]
-line-length = 88
+target-version = ['py36']
 include = '\.pyi?$'
```

Comment:
this was the default (now extended to [include `.ipynb`](https://github.com/psf/black/blob/5e191c29d4492d14f9090a924a77b3dc055443a6/src/black/const.py#L3)), maybe just remove?

Reply:
Good catch -- it's in the black docs so somehow I assumed it was needed.